AUTHORS.md

```diff
@@ -0,0 +1,22 @@
+`romcal` was orginally developed by **Julian Pereira** from Malaysia in July 2014.
```

Comment:
romcal development began by **[Julian Pereira](https://github.com/pejulian)** from Malaysia in July 2014.

- I would not probably style your name in bold.

Reply:
Done!